docs/_docs/01_getting-started/quick-start.md

```
@@ -52,66 +112,82 @@ types of infrastructure code you can test (e.g., Packer, Kubernetes, etc).
  
 {% include examples/explorer.html example_id='terraform-hello-world' file_id='test_code' class='wide quick-start-examples' skip_learn_more=true skip_view_on_github=true skip_tags=true %}
  
-This code does all the steps we mentioned above, including running `terraform init`, `terraform apply`, reading the 
-output variable using `terraform output`, checking its value is what we expect, and running `terraform destroy` 
+This code does all the steps we mentioned above, including running `tofu init`, `tofu apply`, reading the
+output variable using `tofu output`, checking its value is what we expect, and running `tofu destroy`
 (using [`defer`](https://blog.golang.org/defer-panic-and-recover) to run it at the end of the test, whether the test 
 succeeds or fails). If you put this code in a file called `terraform_hello_world_example_test.go`, you can run it by 
 executing `go test`, and you’ll see output that looks like this (truncated for readability):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)